### ISSUE TITLE
Crashes on startup with Forge 14.23.5.2838

### ISSUE THREAD

**Issue by CammiePone** [darkevilmac/MovingWorld#66]
Exactly as the title says. Starting the game in Forge 1.12.2 14.23.5.2838 fails, as MovingWorld causes it to crash, likely due to a change in the files accessed with the mixins. Unfortunately, it's before the game can generate a crash report, so I can't tell for sure.

**Comment by CammiePone** [comment]
Turns out I was wrong, it's actually a crash with Phosphor.

**Comment by Barerock** [comment]
Yup, I'm getting the crash as well. The only way I can think to solve it is to remove one or the other mod. Unfortunate; the author of Phosphor also seems to be unaware of the incompatibility.

**Comment by spikespaz** [comment]
MovingWorld causes the crash because Phosphor tries to load Mixin 0.7.10 and MovingWorld has already loaded 0.7.8.
